sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -1304,14 +1318,19 @@ __ESIMD_API simd<float, SZ> pln(simd<float, 4> src0, simd<float, SZ> src1,
 
   return esimd::saturate<float>(Result);
 }
+/// @} sycl_esimd_math
 
-// bf_reverse
+/// @{
+/// @ingroup sycl_esimd_bitmanip
+
+/// bf_reverse
 template <typename T0, typename T1, int SZ>
 __ESIMD_API simd<T0, SZ> bf_reverse(simd<T1, SZ> src0) {
   simd<unsigned, SZ> Src0 = src0;
   return __esimd_bfrev<unsigned>(Src0.data());
 }
 
+/// bf_reverse
```

Comment:
What's bf and why is it reverse? I think these functions need a todo for a more meaningful description.

Reply:
"bit field reverse". Yes, descriptions are to be added in separate PRs